bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -233,16 +268,21 @@ def _make_load_body(
         project = await self.project()
         url = f'{API_ROOT}/projects/{project}/jobs'
 
-        body = self._make_load_body(source_uris, project)
-        payload = json.dumps(body).encode('utf-8')
+        body = self._make_load_body(
+            source_uris, project, autodetect, source_format, write_disposition,
+        )
+        return await self._post_json(url, body, session, timeout)
 
-        headers = await self.headers()
-        headers.update({
-            'Content-Length': str(len(payload)),
-            'Content-Type': 'application/json',
-        })
+    async def query(
+            self, query: str, session: Optional[Session] = None,
+            write_disposition: Disposition = Disposition.WRITE_TRUNCATE,
```

Comment:
Looks like Google sets the default to `WRITE_EMPTY` -- let's make sure our default matches theirs to avoid any confusion.